stl/inc/memory

```diff
@@ -812,6 +812,22 @@ protected:
         }
     }
 
+    template <class _Ty2>
+    void _Move_construct_avoiding_expired_conversions_from(_Ptr_base<_Ty2>&& _Other) noexcept {

```

Comment:
(Trigger warning: I have an irrational aversion to calling any and everything that involves an rvalue "move meow" when we wouldn't call the lvalue equivalent "copy meow".) Should we name the earlier function `_Copy_construct_meow` for consistency? Should these functions be `_Weakly convert_lvalue_avoiding_meow` and `_Weakly convert_rvalue_avoiding_meow`?

Reply:
`_Weakly_convert_lvalue_avoiding_expired_conversions` and `_Weakly_convert_rvalue_avoiding_expired_conversions` it is.